addins/Cake.Email.yml

```diff
@@ -2,7 +2,7 @@ Name: Cake.Email
 NuGet: Cake.Email
 Assemblies:
 - "/**/Cake.Email.dll"
-Repository: https://github.com/cake-contrib/Cake.Email/
+Repository: https://github.com/cake-contrib/Cake.Email
```

Comment:
@Jericho wondering if this is diff "enough" to warrant an update?

Reply:
We just need to decide what we want in our YAML files and I can make sure to standardize URLs. What do you think our standard should be: with or without slash?